web/themes/contrib/civictheme/civictheme.theme

```diff
@@ -32,6 +32,7 @@ require_once __DIR__ . '/includes/local_tasks.inc';
 require_once __DIR__ . '/includes/libraries.inc';
 require_once __DIR__ . '/includes/map.inc';
 require_once __DIR__ . '/includes/media.inc';
+require_once __DIR__ . '/includes/search.inc';
```

Comment:
Why the shift? Looks like before it was in alphabetical order.

Reply:
@febdao Could you unshift it